stac_mjx/__init__.py

```
@@ -1,5 +1,6 @@
 """This module exposes all high level APIs for stac-mjx."""
 
-from stac_mjx.utils import enable_xla_flags, load_data
+from stac_mjx.op_utils import enable_xla_flags
```

Ruff

`stac_mjx.op_utils.enable_xla_flags` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)